src/components/MultiSelect/SingleSelect.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+
+const useStyles = makeStyles((theme) => ({
+    formControl: {
+        margin: theme.spacing(1),
+        minWidth: 120,
+    },
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+}));
```

Comment:
I would like to keep styling in CSS files. Having styles in CSS and in JS code can make it harder to debug in a later step.